package.json

```diff
@@ -29,7 +29,7 @@
     "test:e2e": "cd tests/playwright && npm run test:e2e",
     "test:e2e:smoke": "cd tests/playwright && npm run test:e2e:smoke",
     "typecheck:shared": "tsc --noEmit --project packages/shared",
-    "typecheck:frontend": "tsc --noEmit --project packages/frontend && cd packages/frontend && pnpm check",
+    "typecheck:frontend": "cd packages/frontend && pnpm check",
```

Comment:
The `check` scripts in `packages/frontend` is the following

https://github.com/podman-desktop/extension-podman-quadlet/blob/abb8d89a00bdf601734e0c96e105bc536eece6b5/packages/frontend/package.json#L15

`svelte-check` cover both `.ts` & `.svelte` file, with Vitest 4, the typing of certain props have been made stricter, failing when running with `tsc` because it is not emitting type for svelte files